lib/wx/src/gen/wxMenuBar.erl

```diff
@@ -253,6 +254,19 @@ oSXGetAppleMenu(#wx_ref{type=ThisT}=This) ->
   wxe_util:queue_cmd(This,?get_env(),?wxMenuBar_OSXGetAppleMenu),
   wxe_util:rec(?wxMenuBar_OSXGetAppleMenu).
 
+%% @doc See <a href="http://www.wxwidgets.org/manuals/2.8.12/wx_wxmenubar.html#wxmenubarmacgetcommonmenubar">external documentation</a>.
```

Comment:
this link is broken, there's no such function on wx 2.8, I think this is a wx 3.0+ addition. I assume it is ok though as wxMenuBar.xml contains the canonical docs.